test-mlflow-tracking/test-mlflow-tracking.ipynb

```diff
@@ -0,0 +1,1906 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ! pip install mlrun==1.6.0-rc11
</pre><p>no need, remove, this anyway will be running only on 1.6.0-rc12 and newer during automatic tests</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>